src/frontend/src/lib/components/hero/Actions.svelte

```diff
@@ -18,6 +18,7 @@
 	import { tokenWithFallback } from '$lib/derived/token.derived';
 	import Receive from '$lib/components/receive/Receive.svelte';
 	import ContextMenu from '$lib/components/hero/ContextMenu.svelte';
+	import Send from '$lib/components/send/Send.svelte';
 
 	export let send = false;
```

Comment:
we still need this option then?

Reply:
nope, but I was going to remove it in another PR

Reply:
Ok. Please add a TODO.

Reply:
Done:

- PR #1951